File: 1.2_30daysofCode/Day6_LetsReview.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = int(input())   
toprint = []
for i in range(n) :
    str = input()
    even,odd = "", ""   
    for j in range(0,len(str),1):
        if(j%2 == 0):
            even+=str[j]   
        
        else:
            odd+=str[j]                    
    
    toprint.append([even,odd]);  

for x in range(len(toprint)):

    print(toprint[x][0],toprint[x][1])

File: 1.2_30daysofCode/Day6_LetsReview.js
function processData(input) {
    //Enter your code here    
    var n = input.split("\n");    
    var toprint = [];
for (let i = 0; i<n[0]; i++ ){
    var str = n[i+1];
    var even = "", odd = "";   
    for(let j = 0; j<str.length; j++ ){
        if(j%2 == 0){
         even+=str[j];   
        }
        else{
            odd+=str[j]
        }            
    }
    toprint.push([even,odd]);  
}
for (let x = 0; x<toprint.length; x++)
{
    console.log(toprint[x][0],toprint[x][1])
}

} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

